mern-ecommerce/controllers/order.js

```diff
@@ -61,3 +61,15 @@ exports.updateOrderStatus = (req, res) => {
     }
   );
 };
+
+exports.deleteOrder = async (req, res) => {
+  try {
+    const order = await Order.findByIdAndRemove(req.params.orderId);
+    if (order) {
+      return res.status(404).json({ message: "Order does not exist" });
+    }
+    res.status(200).json({ message: "Order deleted successfully" });
```

Comment:
The response status code and message should be consistent throughout the application. In this case, the response status code is 200, but the message is "Order deleted successfully". In other parts of the code, the response status code for successful operations is 201. It would be better to use a consistent response status code and message throughout the application.